mongorover/MongoCollection.lua

```diff
@@ -270,15 +270,15 @@ local MongoCollection = {__mode="k"}
 	-- Example usage at @{indexing.lua}.
 	-- @tparam table of fields and their corresponding index types
 	-- @tparam table of options to be used when creating index
-	-- @return string index name on success
+	-- @return string index name
 	function MongoCollection:createIndex(keys,opt)
 	    return self.collection_t:collection_create_index(luaBSONObjects, keys, opt)
 	end
 
 	---
 	-- Delete an index for a specified collection.
 	-- Example usage at @{indexing.lua}.
-	-- @tparam string an index name to be deleted
+	-- @tparam string an index name or original table with indexed fields and their corresponding types
```

Comment:
I would think that retaining that boolean from `mongoc_collection_keys_to_index_string()` would be valuable for lua scripts to validate that index deletion was successful.